stackslib/src/chainstate/stacks/boot/mod.rs

```diff
@@ -160,6 +162,10 @@ pub fn make_contract_id(addr: &StacksAddress, name: &str) -> QualifiedContractId
     )
 }
 
+pub fn make_signers_db_name(signer_set: u32, message_id: u32) -> String {
```

Comment:
I think it might be worthwhile having a helper function to parse out the message id and signer set from the signers_db_name